megamek/src/megamek/client/ui/swing/CustomPilotView.java

```diff
@@ -414,11 +424,17 @@ public int getToughness() {
     }
     
     public String getPortraitCategory() {
-        return portraitDialog.getCategory();
+        if (portraitDialog.getSelectedItem() == null) {
+            return null;
+        }
+        return portraitDialog.getSelectedItem().getCategory();
```

Comment:
```suggestion
        } else {
            return portraitDialog.getSelectedItem().getCategory();
        }
```

Reply:
Eh, this return is the "normal" return value, so I'd rather not put it in an else clause. Purely subjective.